doc/getting_started.rst

```diff
@@ -17,29 +17,162 @@
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
+.. Edited September 25, 2020 - [USA]TechDude
+ 
 Getting Started
 ===============
 
-OpenShot Video Editor is available on most operating systems (including Linux, Mac, and Windows).
-Visit the official download page and grab the latest and greatest version:
-https://www.openshot.org/download/.
-
-Linux
------
-Depending on which Linux distribution you use, there are likely many ways to install OpenShot. The
-easiest way to ensure you have the very latest version, is to download an AppImage from the
-project website (listed above). Once downloaded, right click on the AppImage file, choose
-Properties, and mark the file as **Executable**. Now, go ahead and double click the
-AppImage to launch it.
-
-Mac
----
-Download the DMG file from the project website (listed above), double click it, and then drag
-the OpenShot application icon into your **Applications** shortcut. This is very similar to how most
-other Mac applications are installed. Now launch OpenShot from Launchpad.
-
-Windows
--------
-Download the Windows installer executable from the project website (listed above), double click it,
-and follow the directions on screen. Once completed, OpenShot will be installed and available
-in your Start menu.
+.. image:: images/getting-started.png
+
+You do not need to be a trained videographer to understand how to create videos 
+well.  Simple editing can keep your viewers engaged longer, and add a 
+professional feel, even if you are not a professional video editor.
+
+Basically, video editing is taking footage, cutting it up, removing the 
+pieces you do not want, and keeping the bits you do.  Back in the old days, 
+editing was slicing reels of film and piecing it together.  Thankfully software 
+makes the whole process much more manageable.
+
+There are three main jobs of video editing:
```

Comment:
I'm a little worried about these broad generalizations at the very start of our user guide. Video editing is not the same thing as film editing... and there an infinite # of reasons to utilize video editing. If we want to include a background on non-linear video editing, perhaps we move that to a new section, outside of the "Getting Started". I would suggest this section should be quicker, more focused on downloading, installing, and launching OpenShot.